controller_app/urls.py

```diff
@@ -5,4 +5,5 @@
 
 urlpatterns = [
     path("", views.index),
+    path("<str:backend>/tasks/", views.active_tasks, name="active_tasks"),
```

Comment:
I actually prefer being explicit like this. Using the auth token to determine the backend feels to me like mixing concerns.